do.py

```diff
@@ -1178,6 +1182,54 @@ def authors(ctx_obj, revision_args):
     except subprocess.CalledProcessError:
         print('Error caught: Incorrect revision start or revision end')
 
+"""
+- Currently the run-job/reuse fails (issue https://github.com/scipy/scipy/issues/16009)
+"""
+@cli.cls_cmd('act')
+class Act(Task):
+    """:flashlight: A tool which provides a handy way to run GitHub Actions locally using Docker
+
+    Installation guide :arrow_down:
+    - Install docker desktop/toolkit
+    - Install act
+        * Using homebrew: brew install act
+        * Using bashscript: curl https://raw.githubusercontent.com/nektos/act/master/install.sh | sudo bash
+
+    Options available:
+        * list (lists all the workflows present)
+        * run-job (runs a particular job)
+        * reuse (reuse the containers in act to maintain state)
```

Comment:
The explanation on `reuse` is not clear to me. Does it mean it will not recreate the containers and hence have faster run?

Reply:
I had picked the content for helper texts from this [documentation](http://scipy.github.io/devdocs/dev/contributor/using_act.html?highlight=act), strangely it's showing up as 404.

Reply:
Ok but I personally don't understand this without going to `act`'s doc and reading up more 😅 (404 was temporary)